### ISSUE TITLE
Can't assign roles from the show dashboard

### ISSUE THREAD

**Issue by burlexpo** [bethlakshmi/gbe-divio-djangocms-python2.7#447]
If I go here -- https://www.burlesque-expo.com/scheduling/show_dashboard/4758 there's no way for me to assign Stage Manger, Producer, Tech lead, etc

**Comment by bethlakshmi** [comment]
Yeah - it has to be in the scheduling side...  Special -> Schedule ->
Events.  The dashboard was meant to be available to the already assigned
tech folks, not a tool for assigning them.  Glad to design different next
year.
